src/hotspot/share/interpreter/bytecodeStream.hpp

```diff
@@ -195,7 +195,7 @@ class BytecodeStream: public BaseBytecodeStream {
       int len = Bytecodes::length_for(code);
       if (len == 0) len = Bytecodes::length_at(_method(), bcp);
       if (len <= 0 || (_bci > _end_bci - len) || (_bci - len >= _next_bci)) {
-        raw_code = code = Bytecodes::_illegal;
+        fatal("Should have been caught by verifier");
```

Comment:
Verification may be disabled, This may be a risky change in behaviour.

Reply:
If verification is disabled, this fatal is better than what might come next.  But there may be callers now or in the future that check for Bytecodes::_illegal, so I don't think we should have this assert even though we don't have any code that hits it.